Documentation changes related to GALI-819

shyogesh-sw opened the pull request:
Issue #, if available: GALI-819

Description of changes: Added cross references to installation and troubleshooting in the Packer Readme.

Checklist:

<!-- You can erase any parts of this template not applicable to your Pull Request. -->

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?

<!-- For major releases please provide internal ticket id -->

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

shyogesh-sw committed:
<docs: added hyperlinks in packer readme>

shyogesh-sw committed:
docs: added hyperlinks in packer readme

shyogesh-sw committed:
<docs: added hyperlinks in packer readme>

jn1119 reviewed (commented):
Can you update the commit message to reflect `docs: ` prefix so that semantic pull request check passes. I think currently you have the message under brackets `<>` and that's where the check fails.

edit: Other than that LGTM

shyogesh-sw committed:
docs: added hyperlinks in packer readme
Merge branch 'gali-819' of https://github.com/shyogesh-sw/service-workbench-on-aws into gali-819
